src/olympia/editors/tests/test_views.py

```diff
@@ -2943,7 +2959,7 @@ def test_old_addon_review_action_as_limited_editor(self, mock_sign_file):
         assert self.get_addon().status == amo.STATUS_PUBLIC
         assert mock_sign_file.called
 
-    def test_limited_editor_no_latest_version(self):
-        self.addon.latest_version.delete()
+    def test_limited_editor_no_version(self):
+        self.addon.current_version.delete()
```

Comment:
this is testing a different thing - should be `get_latest_version(amo.RELEASE_CHANNEL_LISTED).delete()` or similar.